pkg/apiserver/api_installer.go

```diff
@@ -108,6 +108,7 @@ func (a *APIInstaller) NewWebService() *restful.WebService {
 	mediaTypes, streamMediaTypes := mediaTypesForSerializer(a.group.Serializer)
 	ws.Produces(append(mediaTypes, streamMediaTypes...)...)
 	ws.ApiVersion(a.group.GroupVersion.String())
+	ws.SetDynamicRoutes(true)
```

Comment:
+1 for putting TPRs in a separate ws, so that there's no way of accidentally removing a route for a built-in API.